src/app/app.component.html

```diff
@@ -1,5 +1,13 @@
-
+ 
  <app-header></app-header>
  <app-section></app-section>
  <router-outlet></router-outlet>
- <app-footer></app-footer>
\ No newline at end of file
+ <app-footer></app-footer>
+  
+ 
+ <ul>
+     <li *ngFor='let recipe of recipes'>
+         <h3>{{recipe.title}}</h3>
+         <p>{{recipe.body}}</p>
+     </li>
+ </ul>
```

Comment:
This list shouldn't be in app.component.html, it should be inside recipe-list.compontent